sync/sync.sh

```diff
@@ -20,4 +20,3 @@ helm dependency update helm/image-distribution-operator
 
 # Patches
 ./sync/patches/helpers/patch.sh
-./sync/patches/manager/patch.sh
```

Comment:
patches are impractical for the manager deployment since we are adding more stuff to it.